src/main/java/nl/novi/techiteasy1121/models/Television.java

```diff
@@ -1,217 +1,230 @@
 package nl.novi.techiteasy1121.models;
 
-import jakarta.persistence.*;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 
 @Entity
+@Table(name = "televisions", schema = "public") // Ensure the schema is set to "public"
 public class Television {
 
-    //  Een entiteit moet een primary key bevatten(id)
     @Id
-    // GeneratedValue betekend dat je deze waarde niet zelf hoeft in te vullen, dit doet Spring Boot voor jou bij het opslaan in de database.
-    @GeneratedValue
-    Long id;
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
```

Comment:
columnName hoef je niet in te vullen als dat hetzelfde is als de naam van de property. In dit geval is de naam dus al "id"